sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -310,18 +304,21 @@ public SignatureSecp256k1 getRealSignature() {
         return null;
     }
 
-    @Override
-    public SignatureSecp256k1 getSignature() {
+    @JsonIgnore
+    public Sign.SignatureData getSignatureData() {
         if (this.isSigned()) {
             SignedRawTransaction stx = (SignedRawTransaction) this.transaction;
-            return new SignatureSecp256k1(
+            return new Sign.SignatureData(
                     stx.getSignatureData().getV(),
                     stx.getSignatureData().getR(),
                     stx.getSignatureData().getS());
         }
         return null;
     }
 
+    public byte[] getV() { return (getSignatureData() != null) ? getSignatureData().getV() : null; }
+    public byte[] getR() { return (getSignatureData() != null) ? getSignatureData().getR() : null; }
+    public byte[] getS() { return (getSignatureData() != null) ? getSignatureData().getS() : null; }
```

Comment:
I think it is never used.
let's remove this methods, of?

Reply:
They are used in json representation, I added a comment